modules/migration-network-traffic-redirection-strategies.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/planning-considerations-3-4.adoc
+// * migration_toolkit_for_containers/network-considerations-3-4.adoc
+
+[id="migration-network-traffic-redirection-strategies_{context}"]
+= Network traffic redirection strategies
+
+After a successful migration, you have to redirect network traffic of your stateless applications from the source cluster to the target {product-title} {product-version} cluster.
+
+All strategies for redirecting network traffic described below are based on these assumptions:
+
+* Applications are deployed on the {product-title} {product-version} cluster. The original application pods are still running on the source cluster.
+* Each application has a route that contains the source cluster hostname.
+* The route with the source cluster hostname contains a CA certificate.
+* If necessary for securing connections made through the source cluster's DNS domain, the {product-title} {product-version} router default certificate includes the source cluster wildcard Subject Alternative Name.
+
+Review the following strategies and select the one that meets your objectives.
+
+* Redirecting all network traffic for all applications at the same time
++
+Change the source cluster wildcard DNS record to point to the {product-title} {product-version} router's virtual IP address (VIP).
```

Comment:
"source cluster wildcard DNS record" is a lot of words to take in. Would be clearer to say "wildcard DNS record of the source cluster".

Global change for this section. Replace product title and version with "target cluster". The name and version do not matter in this context.